app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/forgotpassword/ForgotPasswordScreen.kt

```diff
@@ -87,111 +83,117 @@ fun ForgotPasswordScreen(
-                            .wrapContentHeight()
-                            .fillMaxSize()
-                            .background(AppTheme.colors.white),
-                        horizontalAlignment = Alignment.CenterHorizontally,
-                    ) {
-                        Text(
-                            modifier = Modifier.padding(top = 16.dp),
-                            text = "Password Code",
-                            fontSize = 20.sp,
-                            color = AppTheme.colors.darkPink,
-                            fontWeight = FontWeight(700),
-                            textAlign = TextAlign.Center,
-                            style = AppTheme.typography.bold,
-                        )
-                        BaseTextField(
-                            modifier = Modifier
-                                .padding(top = 20.dp)
-                                .width(140.dp),
-                            textState = code,
-                            textStyle = TextStyle(fontSize = 20.sp, textAlign = TextAlign.Center),
-                            keyboardType = KeyboardType.Number,
-                            focusedContainerColor = AppTheme.colors.backgroundPink,
-                            unfocusedContainerColor = AppTheme.colors.backgroundPink,
-                            shape = AppTheme.shapes.smallRoundedCorners,
-                            onValueChange = { code ->
-                                if (code.length <= codeMaxLength)
-                                    forgotPasswordViewModel.setCode(code)
-                            },
-                        )
-                    }
+                    BaseTextField(
+                        modifier = Modifier.padding(top = 2.dp),
+                        textState = email,
+                        label = stringResource(id = R.string.email),
+                        focusedContainerColor = AppTheme.colors.white,
+                        unfocusedContainerColor = AppTheme.colors.white,
+                        shape = AppTheme.shapes.smallRoundedCorners,
+                        keyboardType = KeyboardType.Email,
+                        imeAction = ImeAction.Done,
+                        errorText = stringResource(id = R.string.register_email_error),
+                        hasError = hasEmailError,
+                        onValueChange = { email ->
+                            forgotPasswordViewModel.setEmail(email)
+                        },
+                    )
                 }
                 AuthButton(
                     modifier = Modifier
                         .padding(horizontal = 32.dp, vertical = 24.dp)
-                        .padding(top = 100.dp)
                         .height(41.dp)
                         .width(200.dp),
-                    backgroundColor = AppTheme.colors.lightPink,
+                    backgroundColor = AppTheme.colors.white,
                     shape = RoundedCornerShape(13.dp),
```

Comment:
You can use RoundedCorderShape defined on AppShapes.

Reply:
Fixed